sympy/geometry/tests/test_polygon.py

```diff
@@ -411,9 +411,15 @@ def test_intersection():
     assert poly1.intersection(RegularPolygon((-12, -15), 3, 3)) == []
 
 
+def test_parameter_value():
+    sq = Polygon((0, 0), (0, 1), (1, 1), (1, 0))
```

Comment:
This test seemed too easy to me, so I tried another one: 
```
q = Polygon((0, 0), (2, 1), (2, 4), (4, 0))
q.parameter_value((4, 0))
```
Got TypeError: cannot determine truth value of Relational

Reply:
Hmm...simplification works but I thought that relationals tried numerical evaluation. In this case you end up with lhs - rhs being a precisionless zero so it fails. I guess simplification will have to be used.

Reply:
I'll add that test, too.